src/txmempool.cpp

```diff
@@ -1150,7 +1149,7 @@ bool CTxMemPool::_LoadMempool(std::function<bool(const CTransactionRef&, int64_t
             if (amountdelta) {
                 PrioritiseTransaction(tx->GetHash(), amountdelta);
             }
-            if (nTime > nNow - nExpiryTimeout) {
+            if (nTime > TicksSinceEpoch<Seconds<int64_t>>(now - m_expiry)) {
```

Comment:
05ff88acdfc1b777e7d6e53a2975e18ead6f35b1:

What do you think about `if (std::chrono::seconds{nTime}  > now - m_expiry) {` ?

Reply:
I tried it, and I think the only thing that'll compile is:

```cpp
            if (NodeSeconds{std::chrono::seconds{nTime}} > now - m_expiry) {
```

Which doesn't seem better than what we have now

Reply:
I see. Is there any reason why you need `Seconds<int64_t>`? `std::chrono::seconds` should already be of the same width.

Reply:
From reading the docs, the representation for `std::chrono::seconds` seems to only need to be "A signed integral type of at least 35 bits", so there might be platforms/stdlib's that don't use `int64_t` and I thought it'd be better to be explicit about it.

Reply:
Is there anything that requires this code part to be explicit about it? Does it protect against something?

Reply:
It doesn't really, happy to change it to `std::chrono::seconds` if you want me to!

Reply:
If there is no reason to change util/time.h, then I think the preference is to not change it.

Reply:
Agreed! Done in ddf4920df58bc6f84297abc9b31e5830237ea514